tests/services/audio/test_audio_transcriber_service.py

```
@@ -298,23 +301,17 @@ def test_cached_model(self, transcriber):
     def test_import_error(self, transcriber):
-        def fake_import(name, *args, **kwargs):
-            if "faster_whisper" in name:
-                raise ImportError("no faster_whisper")
-            return original_import(name, *args, **kwargs)
-
-        import builtins
-
-        original_import = builtins.__import__
-        with patch("builtins.__import__", side_effect=fake_import):
+        with patch("file_organizer.services.audio.transcriber._FASTER_WHISPER_AVAILABLE", False):
             with pytest.raises(ImportError, match="faster-whisper is required"):
```

Ruff (0.15.9)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)